test-sweet.js

```diff
@@ -28,10 +28,16 @@ const method = (name, expectations) => {
   expectations()
```

Comment:
I think we can do something similar to our matchers here so we can have several different descriptors